### PR TITLE
Changes for prerender release

### PR DESCRIPTION
**Pull request by dac09**
**As discussed for 0.26 release**

- [x]  [Danny] Rename index.js → App.js in template
    - [x]  Update auth setup command
    - [x]  Update tests (snapshots should NOT change, apart from rename)
- [x]  [Danny] Update prerender code
- [x]  [Danny] Update webpack config for hidden index
- [x]  [Danny] Update `rw setup custom-entry`
- [x]  [Danny] Remove the `experimentalPrerender` flag from TOML

### CODE REVIEW

File: packages/cli/src/commands/setup/auth/auth.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ const removeOldAuthProvider = async (content) => {
 
 // check to make sure AuthProvider doesn't exist
 const checkAuthProviderExists = () => {
-  const content = fs.readFileSync(WEB_SRC_INDEX_PATH).toString()
+  const content = fs.readFileSync(WEB_SRC_APP_PATH).toString()
```
Comment (peterp) [inline packages/cli/src/commands/setup/auth/auth.js:131]:
```suggestion
  const content = fs.readFileSync(WEB_SRC_APP_PATH, 'utf8')
```

Comment (peterp) [reply packages/cli/src/commands/setup/auth/auth.js:131]: If the encoding is specified then this will return a string rather than a buffer.

**Review by peterp — APPROVED** [review]
Looks good to me, just added a note about specifying the encoding, but not an issue.

File: packages/cli/src/commands/setup/auth/auth.js (hunk 1 above)
Comment (dac09) [reply packages/cli/src/commands/setup/auth/auth.js:131]: Let me change these afterwards @peterp just to minimise changes if its ok?

**Comment by thedavidprice** [comment]
let's do this 🚀

**Pull request merged**